### PR TITLE
Update Code.string_to_quoted columns default value

### PR DESCRIPTION
**Pull request by lukaszsamson**
Default value changed in https://github.com/elixir-lang/elixir/pull/12941

### CODE REVIEW

File: lib/elixir/lib/code.ex [hunk 1]
```diff
@@ -1095,7 +1095,7 @@ defmodule Code do
       Defaults to 1.
 
     * `:columns` - when `true`, attach a `:column` key to the quoted
-      metadata. Defaults to `false`.
+      metadata. Defaults to `true`.
```
Comment (whatyouhide) [inline lib/elixir/lib/code.ex:1098]: Should we document that the default was different up to Elixir 1.15?

Comment (lukaszsamson) [reply lib/elixir/lib/code.ex:1098]: We could. We could also put it in breaking changes - some libraries pattern match on AST metadata and break when compiling with `:columns` - at least 2 such cases were reported to elixir-ls

Comment (josevalim) [reply lib/elixir/lib/code.ex:1098]: We didn't change the value here. We changed it on parser_options which I already fixed in another commit. :) This one still remains the same!

Comment (lukaszsamson) [reply lib/elixir/lib/code.ex:1098]: Oh, I didn't notice the defaults are now different in those 2 cases. @josevalim what about my comment on documenting the breaking change?

**Pull request closed without merging**